### DISCUSSION TITLE
Ambient Lockout Settings

### DISCUSSION THREAD

**Discussion by imcintyr** [PeteRager/lennoxs30#297] [Q&A]
Hi there, we have a new heat pump (EL22XPV) and solar panels as of last summer/fall. I am noticing that there are times I would like to adjust the ambient lockout temperature settings based on the amount of cloud cover during the day. Using our weather station to monitor the amount of light coming through or perhaps the power going through the inverters since they are integrated into HA is the idea.

From what I have read in the README and any discussion thread containing the word "ambient", the ambient lockout is only shown as an alert and the info is only on/off.  No adjustments can be made to the temp setting through the HA/Lennox30 integration that I can find. Is this correct? If yes, is there a workaround I am not seeing here?

Thanks for any help!

**Comment by PeteRager** [comment]
I like it.  Great use case.

There should be equipment parameters to control those, you may need to enable those in the integration configuration.  I see these three that should allow tuning this parameters.

Balance Point Control
Low Balance Point
HP heating Lockout Time

You should also be able to find them on the s40 panel under advanced equipment settings. I would try changing them from there and determine how/when they take affect.  When you update them, they should also update in HA.  Then try changing them in HA.  There is a parameter safety switch you need to turn off before you can set those in HA.

Some equipment parameters require a sequence of steps to make them active.  There is a sample script that works for setting the CFM fan parameter.

https://github.com/PeteRager/lennoxs30/tree/master/samples/set_fan_cfm

I would try it first by just setting the parameter and see if it takes effect.  If it doesn't then try the script and have it set the low balance point instead of fan CFM.

when you get it working, post the sample.

**Reply by imcintyr** [reply]
The parameters are working as described. They update between the Lennox app and my desktop faster than I can move my eyes between them so that's a good sign.

The weather station is proving more fussy to get the data flowing but I'll keep at it.

Thanks again for the help and I'll post this Rube Goldberg special once it's up and running!

**Comment by graveland** [comment]
Posting to an old thread, but I have the same desire: HA can tell me if I have a solar surplus, and I'd like to use that to make decisions about using the HP or gas. I see I can change the balance point, but it only lets me change it within a very small window (0-15°F). Do you have suggestions of other ways I could tell the thermostat to use or not use the HP? What do `Low Balance Point` and `HP heating Lockout Time` do?

**Reply by PeteRager** [reply]
Put it into emergency heat mode.

https://github.com/PeteRager/lennoxs30?tab=readme-ov-file#supported-operations

**Reply by PeteRager** [reply]
That's actually the inverse answer.  Apologies.  The Lennox installer docs have good descriptions of the parameter behavior

**Comment by graveland** [comment]
Regarding the select for emergency heat: when I set that, about 20s later it reverts to its previous setting (heat_cool). Does away mode impact how this is handled?
